File: javascript/matrix/matrix.js

export class Matrix {

  constructor(inputString) {

    this._rows = inputString.split("\n").map(row => {

      const cells = row.split(' ')

      return cells.map(cell => {
        return parseInt(cell)
      })

    })

    this._columns = []

    this._rows.forEach((row) => {

      row.forEach((cell, i) => {

        if (this._columns[i] === undefined)
          this._columns[i] = []

        this._columns[i].push(cell)

      })

    })

  }

  get rows() {
    return this._rows
  }

  get columns() {
    return this._columns
  }

}

File: python/matrix/matrix.py
class Matrix:

    rows = []
    columns = []

    def __init__(self, matrix_string):

        print("ok: ", self.rows)

        self.rows = matrix_string.split('\n')
        
       
        # self.columns = 

        print("rows: ", self.rows)
        pass

    def row(self, index):



        print("index: ", index - 1)
        print("thing: ", self.rows[index -1])
        # return self.rows[index]
        return  [int(i) for i in str(12345)]

    def column(self, index):
        pass
